app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,44 @@
+<h1> Welcome to your to-do list! Your tasks are: </h1>
+
+<div class = "tasklist">
+  <% @task_list.each do |task| %>
+  <% if @task_list == nil %>
```

Comment:
For here you can just check if @task_list == nil instead of looping through the list! It doesn't really matter here, but if your task list was very large, it would slow performance to loop through each one when you don't need to!